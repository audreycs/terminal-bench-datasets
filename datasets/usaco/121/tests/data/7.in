110011110001101000111111101010011010001011000010111001010001010001001100000000101001110000011000000100111010110000100101010101111011000110100110010011011101001011001101011010101111101110000000010010000111111001110011010101111100100010001110111001010100111001101000100111010111111010111110110010111101110001011111001101010001101101111110001100110011010110101010001111111111000010100011010100010010010100111101001100101010101101111011111011111110101100100011000000100011111001111000010001100000011000001110100110111101100000011110110011100111111000101111101011010100101100111101000100001000011101101111100000000110000011010111010110010111110111101100000010101000110001100100111001100001010011010101111100111011110001111010111100001101100101111101111111011010101101011101000011
