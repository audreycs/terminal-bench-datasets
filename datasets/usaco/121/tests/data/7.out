1101110000001011111000111010010000111100111011110001001101100101100100001100001011000101110110011001010011100110111001111010110100101100110000001011001010101111111110100100000110101011001110000100110100000110010110101000110101000101000101111101001110100011010011110010011101010110101010101011100010011010001001010010100001101101001101100001011001101001000001001110001111101111101011011000011000110111100100010000010111010110001100111010111011101001110101010011001001100010010111111100101001100110011011111000010110010101101000001011101101100110000100101010100000011111111100001110000110001111111001100111100001100110111001001100111100010101110010101100101100110101001010110011010001110110001000110101001011111000010000101001111111100111000101011101110110000110000100101101110011
